apps/web/src/components/editor/panels/timeline/timeline-toolbar.tsx

```
@@ -142,61 +252,14 @@ function ToolbarLeftSection() {
-
-				<ToolbarButton
-					icon={
-						<HugeiconsIcon
-							icon={isSelectedSourceAudioSeparated ? Unlink02Icon : Link02Icon}
+				{Object.values(ToolTipShortcuts).map((item)=> (
+					<ToolbarButton
+						icon={item.icon}
+						disabled={item.disabled}
+						tooltip={item.tooltip}
+						onClick={item.onClick}
 						/>
```

Biome (2.4.11)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)